networking/ovn_kubernetes_network_provider/configuring-egress-traffic-for-vrf-loadbalancer-services.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="configuring-egress-traffic-loadbalancer-services"]
+= Configuring an egress service
+include::_attributes/common-attributes.adoc[]
+:context: configuring-egress-traffic-loadbalancer-services
+
+toc::[]
+
+As a cluster administrator, you can configure egress traffic for pods behind a load balancer service by using an egress service.
+
+:FeatureName: Egress service
+include::snippets/technology-preview.adoc[]
+
+You can use the `EgressService` custom resource (CR) to manage egress traffic in the following ways:
+
+* Assign a load balancer's ingress IP address as the source IP address for egress traffic for pods behind a load balancer service.
```

Comment:
nit: a service of type load balancer's IP (the fact that it is related to ingress is implicit)